### DISCUSSION TITLE
2023년 02월 24일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#2] [General]
# 2023년 02월 24일

LOG 너무 졸려서 어제 TIL도 못 쓰고 자버렸다. 그나마 오늘은 할 일이 많이 없어서 어제 TIL도 적었다. 다행히 기억이 좀 나서 적을 수 있었던 게 좋았다. 포비와 1:1 면담(?)을 했다. 간단하게 내 소개와 내가 왜 우테코에 뽑혔을지에 대…

https://cyma-s.github.io/2023-02-24/

**Comment by poi1649** [comment]
```우테코에서 지내면서 알게 된 것은 잘못된 것은 없다는 것이다```
공감 드립니다. 베로 화이팅!
